manifests/repo.pp

```diff
@@ -62,7 +64,7 @@
         yumrepo { 'zabbix':
           name     => "Zabbix_${majorrelease}_${facts['os']['architecture']}",
           descr    => "Zabbix_${majorrelease}_${facts['os']['architecture']}",
-          baseurl  => "https://repo.zabbix.com/zabbix/${zabbix_version}/rhel/${majorrelease}/\$basearch/",
+          baseurl  => pick($repo_location, "https://repo.zabbix.com/zabbix/${zabbix_version}/rhel/${majorrelease}/\$basearch/"),
```

Comment:
could you use a selector statement instead of `pick()`? That would allow us to stay with `''` as default for `$repo_location`, so it wouldn't be a breaking change.